lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -144,9 +149,23 @@ export class CryptoOps implements ICrypto {
      * Removes all cryptographic keys from IndexedDB storage
      */
     async clearKeystore(): Promise<boolean> {
-        const dataStoreNames = Object.keys(this.cache);
-        const databaseStorage = this.cache[dataStoreNames[0]];
-        return databaseStorage ? await databaseStorage.deleteDatabase() : false;
+        try {
+            this.logger.verbose("Deleting in-memory and persistent asymmetric key stores");
+            await this.cache.asymmetricKeys.clear();
+            this.logger.verbose("Successfully deleted asymmetric key stores");
+            this.logger.verbose("Deleting in-memory and persistent symmetric key stores");
+            await this.cache.symmetricKeys.clear();
+            this.logger.verbose("Successfully deleted symmetric key stores");
+            return true;
+        } catch (e) {
+            if (e instanceof Error) {
+                this.logger.error("Clearing keystore failed with error: ", e.message);
```

Comment:
This passes `e.message` as correlationId, which I don't think you meant. 

```suggestion
                this.logger.error(`Clearing keystore failed with error: ${e.message}`);
```